packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/FIAPPaymentQueueDeleteTests.m

```diff
@@ -69,15 +69,19 @@ - (void)testShouldContinueTransaction {
 
 - (void)testShouldContinueTransaction_should_default_to_yes {
   if (@available(iOS 13.0, *)) {
+    TestMethodChannel *mockChannel = [[TestMethodChannel alloc] init];
     FIAPPaymentQueueDelegate *delegate =
-        [[FIAPPaymentQueueDelegate alloc] initWithMethodChannel:self.channel];
+        [[FIAPPaymentQueueDelegate alloc] initWithMethodChannel:mockChannel];
 
-    OCMStub([self.channel invokeMethod:@"shouldContinueTransaction"
-                             arguments:[FIAObjectTranslator getMapFromSKStorefront:self.storefront
-                                                           andSKPaymentTransaction:self.transaction]
-                                result:[OCMArg any]]);
+    mockChannel.invokeMethodChannelWithResultsStub =
+        ^(NSString *_Nonnull method, id _Nonnull arguments, FlutterResult _Nullable result) {
+          XCTAssertEqualObjects(method, @"shouldContinueTransaction");
+          XCTAssertEqualObjects(arguments,
+                                [FIAObjectTranslator getMapFromSKStorefront:self.storefront
+                                                    andSKPaymentTransaction:self.transaction]);
+        };
 
-    BOOL shouldContinue = [delegate paymentQueue:OCMClassMock(SKPaymentQueue.class)
+    BOOL shouldContinue = [delegate paymentQueue:[[SKPaymentQueueStub alloc] init]
```

Comment:
can you keep the same naming convention for these types? (FooStub vs TestFoo)

Reply:
renamed